funnel/views/project.py

```diff
@@ -389,7 +390,7 @@ def save(self):
     @route('admin', methods=['GET', 'POST'])
     @render_with('admin.html.jinja2')
     @lastuser.requires_login
-    @requires_permission('checkin_event')
+    @requires_roles({'profile_admin', 'project_usher'})
```

Comment:
Why does an usher have access to something named `admin`? Which of these is incorrect? The role or the view name?